sdks/java/io/debezium/src/test/java/org/apache/beam/io/debezium/DebeziumIOMySqlConnectorIT.java

```diff
@@ -98,19 +122,22 @@ private void monitorEssentialMetrics() {
           rs.close();
           Thread.sleep(4000);
         } else {
-          throw new IllegalArgumentException("OIOI");
+          throw new IllegalArgumentException(
+              "Illegal Argument Exception in monitorEssentialMetrics.");
         }
       }
-    } catch (InterruptedException | SQLException ex) {
-      throw new IllegalArgumentException("Oi", ex);
+    } catch (SQLException ex) {
+      LOG.error("SQL error in monitoring thread. Shutting down.", ex);
```

Comment:
The exception is suppressed here. Is it intended? The test will pass even with exception here.

Reply:
Was not intended, thank you for the catch, it's thrown now and causing the test to fail.